core/src/test/java/org/elasticsearch/rest/BaseRestHandlerTests.java

```diff
@@ -160,8 +160,8 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
         final HashMap<String, String> params = new HashMap<>();
         params.put("format", randomAsciiOfLength(8));
         params.put("filter_path", randomAsciiOfLength(8));
-        params.put("pretty", randomAsciiOfLength(8));
-        params.put("human", randomAsciiOfLength(8));
+        params.put("pretty", "");
```

Comment:
Correct; the test did not care about that but the handler still parses the value so I think it makes sense to randomize it.